packages/backend/src/queryBuilder.ts

```diff
@@ -175,10 +175,8 @@ export const buildQuery = ({
             explore,
             compiledMetricQuery,
         );
-        return [
-            ...acc,
-            `  ${metric.compiledSql} AS ${fieldQuoteChar}${alias}${fieldQuoteChar}`,
-        ];
+        const renderedSql = `  ${metric.compiledSql} AS ${fieldQuoteChar}${alias}${fieldQuoteChar}`;
+        return acc.includes(renderedSql) ? acc : [...acc, renderedSql];
```

Comment:
@owlas  is there a chance that the same sql chunk will be repeated in some other part of the generated sql?

Reply:
I guess this is localised to a one where group